src/icons/SmallSeal.tsx

```diff
@@ -0,0 +1,92 @@
+import { displayTo } from 'helpers/visibilityClassnames'
+import { stroke } from 'classnames/tailwind'
+
+const pathColor = (connected?: boolean) =>
```

Comment:
You're doing this function invocation like 15 times in this icon, so it will check for `connected` 15 times, maybe we should put this `const` into the component and use as `const pathColor = classnames(conntected ? .. : ..)`